modules/social_features/social_event/social_event.module

```diff
@@ -848,7 +915,8 @@ function social_event_entity_operation_alter(array &$operations, EntityInterface
  */
 function social_event_node_update(EntityInterface $entity) {
   if ($entity->getType() === 'event') {
-    $enroll_state_original = $entity->original->get('field_enroll_method')->getString();
```

Comment:
@dmytriikaiun Looks like those changes not relevant and were added from code formatting of your IDE.